react/features/recording/components/AbstractRecordingLabel.js

```diff
@@ -84,12 +84,30 @@ export default class AbstractRecordingLabel
         super(props);
 
         this.state = {
-            staleLabel: false
+            staleLabel: true
```

Comment:
Is this change intentional?

Reply:
yes. this is actually the fix. For the main jitsi meet screen, the labels are only instantiated once, for carmode, each time the carmode window opens - so when the carmode window opens, if it finds staleLabel as false, it will show the labels until the 10 seconds timeout passes. I tested and this keeps the correct 10 seconds wait behavior for main jitsi meet screen and shows the status real-time in Carmode.